### DISCUSSION TITLE
How to ignore specific workflow rules, field updates, etc.

### DISCUSSION THREAD

**Discussion by janjulian** [scolladon/sfdx-git-delta#736] [Q&A]
I can't seem to find examples of this, nor discussions here that suggest whether or not it's possible. 

It does seem to be supported based on the small note on the README.md under [Explicitly including specific files for inclusion or destruction regardless of diff](https://github.com/scolladon/sfdx-git-delta#explicitly-including-specific-files-for-inclusion-or-destruction-regardless-of-diff)

> /!\ In order to work properly with metadata contained inside files (Labels, Workflow, MatchingRules, etc) the local repo must have the full historic.

**Comment by scolladon** [comment]
Hi @janjulian 

It is possible to ignore every metadata as file.
But not possible for now to ignore metadata as children of a file (like workflow as you pointed it).

If you think it is important you can create a [feature request](https://github.com/scolladon/sfdx-git-delta/issues/new?assignees=scolladon&labels=enhancement&projects=&template=enhancement.md&title=%5BFEATURE+NAME%5D) and we will prioritize it 👍
